WordPress/Classes/ViewRelated/Aztec/New Group/AztecVerificationPromptHelper.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  AztecVerificationPromptHelper.swift
+//  WordPress
+//
+//  Created by Jan Klausa on 05.10.17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import UIKit
+
+internal class AztecVerificationPromptHelper: NSObject {
+
+    private let managedObjectContext: NSManagedObjectContext
+    private let accountService: AccountService
+    private let wpComAccount: WPAccount
+
+    private weak var displayedAlert: FancyAlertViewController?
+    private var completionBlock: AztecVerificationPromptCompletion?
+
+    typealias AztecVerificationPromptCompletion = (Bool) -> ()
+
+    init?(managedObjectContext: NSManagedObjectContext, for account: WPAccount?) {
```

Comment:
I think this `init` could be improved a bit by just passing the account.  An early `guard` statement could handle the case the `account` was `nil` and then the context could be obtained from calling `account.managedObjectContext`.  The early check would the extra work of instantiating `accountService` when the accout is nil.
Just a thought.

Reply:
Rewritten :)